### ISSUE TITLE
Use of angular 2 typescript 

### ISSUE THREAD

**Issue by snahmad** [mongoose-os-apps/wifi-setup-web#4]
Can I use my own custom web application to scan wifi and configure device with some other web pages.

I am using angular 2 typescript . Do I need to use JavaScript or can i use C/C++ to scan Access point and get list.

**Comment by tripflex** [comment]
Yes you can, all this example app does is show how to use AJAX to make an RPC call to setup the device.

You can write your own JS to make the RPC call to get access points (see other example apps), to configure the device, or you can call the native C code to get access point lists.

Make sure to read the documentation, there's libs available for almost everything you would need to do over RPC .. and if not, it's very easy to add your own RPC endpoints:
https://mongoose-os.com/docs/userguide/rpc.md
